src/gradualizer_type.erl

```diff
@@ -338,6 +345,8 @@
                           [af_lit_atom('is_subtype') |
                            [af_type_variable() | abstract_type()]]}. % [IsSubtype, [V, T]]
 
+-type gr_any_type() :: {'type', anno(), 'any', []}.
```

Comment:
Why do we need to add this?

I think it's covered by this definition:

```erlang
-type af_predefined_type() ::
        {'type', anno(), type_name(),  [abstract_type()]}.
```

Reply:
It's not necessarily needed. I used it [here](https://github.com/josefs/Gradualizer/blob/5d9bfccaf4a16972f9ee28c1f7e798cd34642411/src/typechecker.hrl#L8) to emphasise that `fenv` carries either a spec or simply `any()` (wrapped in a list) for each function (as it wasn't very apparent to me). We could inline it there like this:
```erlang
-record(env, {fenv              = #{}   :: #{{atom(), arity()} =>
                                                  [gradualizer_type:af_constrained_function_type()]
                                                | [{'type', gradualizer_type:anno(), 'any', []}]
                                            },
```
but I'm not sure whether it's better. What would you do?

Reply:
OK. I'm fine with any. :-)